pom.xml

```diff
@@ -215,12 +216,12 @@
                     </configuration>
                     <executions>
                         <execution>
-                            <phase>compile</phase>
+                            <phase>clean</phase>
```

Comment:
I needed that ANTLR executes before compile and I also needed that the formatter check executes before the ANTLR generation, so the ANTLR gen executes in the validate phase, and the formatter check in the clean phase.